src/schema.js

```diff
@@ -89,8 +89,6 @@ const schema = {
       nodes: [
         {
           match: [
-            { object: 'text' },
-            { type: 'link' },
```

Comment:
Shouldn't this include lists as well?

Reply:
I believe so, but I think that should be left for another issue when we add support for nested lists.